src/main/java/de/unistuttgart/overworldbackend/data/enums/BarrierType.java

```diff
@@ -0,0 +1,6 @@
+package de.unistuttgart.overworldbackend.data.enums;
+
+public enum BarrierType {
+    worldBarrier,
+    dungeonBarrier
```

Comment:
Constants are written in `UPPER_CASE`.
If you need this specific name (which I hope you don't), override `toString`.